open-apparel-registry/open-apparel-registry
# Rename  "Account Name" to "Contributor Name"

## Overview

Adjust registration form field labels and descriptions in order to clarify during new registrations that "Account Name" and "Account Description" will be publicly listed as the "Contributor Name" in the facilities dropdown and the "Contributor Description" on the profile page.

Connects #441 

## Demo

![Screen Shot 2019-04-01 at 11 26 20 AM](https://user-images.githubusercontent.com/4165523/55339812-6855c580-5471-11e9-80ad-bc0a80a780b9.png)

## Testing Instructions

- serve this branch
- visit the registration form
- verify that Account Name and Account Description have the new labels & descriptions depicted above
- complete and submit the form and verify that registration still works

## Checklist
